tools/azure-sdk-tools/devtools_testutils/azure_recorded_testcase.py

```diff
@@ -261,3 +189,107 @@ def generate_sas(self, *args, **kwargs):
         sas_func_pos_args = args[1:]
         token = sas_func(*sas_func_pos_args, **kwargs)
         return token
+
+def get_credential(is_live=True, **kwargs):
```

Comment:
I assume that `is_live` was added as a parameter here so that it could accept the value of `self.is_live` from `AzureRecordedTestCase.get_credential`, but we can actually use the module-level `is_live` method from `devtools_testutils.helpers`: https://github.com/Azure/azure-sdk-for-python/blob/9f872dd538da168b638deb9c5de6ad61ef0aa671/tools/azure-sdk-tools/devtools_testutils/helpers.py#L76-L77

If we replace `is_live` references with `is_live()` in this method, we can clean up the signature 🙂